numba/tests/test_parallel_backend.py

```diff
@@ -897,6 +896,27 @@ def run_cmd(cmdline):
             print("OUT:", out)
             print("ERR:", err)
 
+    def test_lazy_lock_init(self):
+        # checks based on https://github.com/numba/numba/pull/5724
+        # looking for "lazy" process lock initialisation so as to avoid setting
+        # a multiprocessing context as part of import.
+        for meth in ('fork', 'spawn', 'forkserver'):
+            # if a context is available on the host check it can be set as the
+            # start method in a separate process
+            try:
+                multiprocessing.get_context(meth)
+            except ValueError:
+                continue
+            cmd = ("import numba; import multiprocessing;"
+                   "multiprocessing.set_start_method('{}');"
+                   "print(multiprocessing.get_context().get_start_method())")
+            cmdline = [sys.executable, "-c", cmd.format(meth)]
+            out, err = self.run_cmd(cmdline)
+            self.assertIn(meth, out)
+            if self._DEBUG:
```

Comment:
Perhaps the following print statements should be placed before the call to `assertIn`? This way, the values will be printed also if the test fails?

Reply:
Good point, done in 
2f8c737

Reply:
Great!